### PR TITLE
Fix the problem that CI can not feel the failure of test case

### PR DESCRIPTION
**Pull request by wyunchi-ms**
## Checklist

- [x] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: tools/TestNet472Modules.ps1 [hunk 1]
```diff
@@ -32,14 +34,19 @@ $rmItems | %{`
...
-    -WorkingDirectory $testDir `
-    -NoNewWindow `
-    -ArgumentList $_.FullName, $testConfig, '-trait "AcceptanceType=CheckIn"', '-notrait "RunType=DesktopOnly"', '-notrait "RunType=CoreOnly"', "-xml $logPath" `
+	  # xunit.console will not throw an exception when the tests failed. So we should handle the exitcode by adding option PassThru
+	  $process = Start-Process -FilePath $newExecPath `
+		-Wait `
+		-WorkingDirectory $testDir `
+		-NoNewWindow `
+		-ArgumentList $_.FullName, $testConfig, '-trait "AcceptanceType=CheckIn"', '-notrait "RunType=DesktopOnly"', '-notrait "RunType=CoreOnly"', "-xml $logPath"  -PassThru 
+	  if ($process.ExitCode -ne 0) {
```
Comment (markcowl) [inline tools/TestNet472Modules.ps1]: nice

**Comment by markcowl** [comment]
@wyunchi-ms  You might want to pull in my PR so that we can get past the failure int he Accounts tests.

Is there any way we can continue running tests after a particular test failure, so we can catch all test failures at once?

**Comment by markcowl** [comment]
@wyunchi-ms, closing in favor of #10129, which also has this your commits (and fixes the test issue).  Nice work!

**Pull request closed without merging**